pages/projects.js

```diff
@@ -0,0 +1,40 @@
+import ProjectData from '../assets/data/projects.json';
+import { Row, Col, Container, Card } from 'react-bootstrap';
+import Image from "next/image";
+
+function Project(props) {
+    const {img, name, desc, src} = props
+    return (
+        <Col lg={6} md={12} className="mb-5 mt-5">
+            <Card style={{ width: 'auto', height: '100%', borderRadius:"30px", boxShadow:"2px 4px 10px" }}>
+                {/* <Card.Img variant="top" src={img_src} /> */}
+                <Image
+                style={{borderTopLeftRadius:"30px", borderTopRightRadius:"30px"}}
+                height="300"
+                width="300"
+                src={img}/>
```

Comment:
Since we are statically hosting this (not using a nextjs server), I think we'll need to use the regular `<img>` tag.

`<Image>` is a special nextjs thing that dynamically sizes the image based on each request.